### DISCUSSION TITLE
update to 3.10.5 not working correctly

### DISCUSSION THREAD

**Discussion by frostypooky** [OnionUI/Onion#539] [General]
Hello - i tried to update my miyoo mini to onion OS v3 today from a previous version of onion, the update goes well, then after restart i get the miyoo boot up, the onion boot up - then nothing, just a black screen, ill even get the onion shut down screen on power down. ive tried re-installing the update several times, any suggestions?

**Comment by dwmccoy** [comment]
Update for 3.10.5 requires that you already have version 3.10.x or higher installed.  You can't patch versions earlier than 3.10.x to 3.10.5.

**Reply by frostypooky** [reply]
thank you for the insight - so would i just have to back up saves, format the card, then do a fresh install?

**Reply by dwmccoy** [reply]
Yes.  Depending on your last version of Onion OS installed due to default emulator changes, some save states may not work.

**Reply by frostypooky** [reply]
so i tried formatting the card and doing a fresh install - still getting the same issue. im following the install guide, could there be something else im missing?

**Reply by Sichroteph** [reply]
You did not read the instructions and did not to update your firmware.

**Reply by frostypooky** [reply]
> You did not read the instructions and did not to update your firmware.

yeah i misunderstood the instructions - i thought that only needed to be done for a fresh install, got it working now, it was indeed the firmware
